go.mod

```diff
@@ -2,6 +2,9 @@ module github.com/snapcore/snapd
 
 go 1.13
 
+// maze.io/x/crypto/afis imported by github.com/snapcore/secboot/tpm2
+replace maze.io/x/crypto => github.com/pedronis/maze.io-x-crypto v0.0.0-20190131090603-9b94c9afe066
```

Comment:
I may miss some context here (sorry!) but shouldn't we move this to a shared location like `github.com/snapcore/maze.io-x-crypto` instead ?

Reply:
Fixed. Thanks for the repo help.